app-creator/src/redux/actions.ts

```diff
@@ -57,7 +128,7 @@ export const setSelectedTab = (index: Tab): SetSelectedTabAction => {
   return {
     type: SET_SELECTED_TAB,
     payload: {
-      index,
+      selectedTab: index,
```

Comment:
Does this payload not need an `eventType`?

Reply:
We're using eventType to distinguish between the global element state. In this case, since we're not changing the element, we don't need eventType. We could send EventType.none to make sure that the state returns to default.